.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,21 @@
-<!--- Thanks for sending a pull request! Please make sure to read the contribution guidelines, then fill out the blanks below before requesting a code review. -->
+<!--- Thanks for sending a pull request!
+
+Please make sure to read the contribution guidelines, then fill out the blanks below before requesting a code review.
+
+Name your Pull Request with one of the following prefixes, e.g. "feat: add support for PyTorch", to indicate the type of changes proposed. This is based on the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary).
```

Comment:
Since this is a comment i think comment format won't show up, so the links would be just fine

Reply:
I think it's still better to use the markdown format so people immediately know it's the conventional commits spec